config/webpack.prod.js

```diff
@@ -0,0 +1,65 @@
+const path = require("path")
+const htmlWebpackPlugin = require("html-webpack-plugin")
+const miniCSSExtractPlugin = require('mini-css-extract-plugin')
+const optimizeCSSAssets = require('optimize-css-assets-webpack-plugin')
+const compressionPlugin = require('compression-webpack-plugin')
+const globalVariables = require('./globalVariables')
+const babelConfig = require('./babel-config')
+const stylusConfig = require('./stylus-config')
+
+const prodConfig = env => ({
+  entry: {
+    // main: ["./src/main.js"]
```

Comment:
:scissors:

Reply:
Yes, I'll remove! But, i'm not so sure that we can already get rid of the Babel's polyfill yet or not!!!!

I'll take a look!!